acoular/process.py

```diff
@@ -332,7 +332,7 @@ class SampleSplitter(InOut):
     _buffer_overflow = Bool(False)
 
     # Helper Trait holds source generator
-    _source_generator = Trait()
+    _source_generator = Any()
```

Comment:
This should be `Instance(LockedGenerator)`

Reply:
I'll change this.